docs/references/chrome-extension/configure-consistent-crx-id.mdx

```diff
@@ -1,90 +1,112 @@
 ---
-title: Configure a consistent CRX Id
-description: Learn how to configure a consistent CRX Id in your Chrome Extension for development and production.
+title: Configure a consistent CRX ID for your Chrome Extension
+description: Learn how to configure a consistent CRX ID in your Chrome Extension.
 ---
 
-Your Chrome Extension has its own unique CRX Id. This key can change while you are developing, unless you take steps to configure the extension. This guide will cover those steps so that your extension will have a stable, unchanging key.
+Chrome Extensions have a unique CRX ID that rotates by default, which can cause errors with the Clerk integration. This guide demonstrates how to configure a consistent CRX ID so that your extension will have a stable, unchanging key.
+
+There are two ways to configure a consistent CRX ID:
+
+1. [For a new extension](#for-a-new-extension).
+1. [For an extension uploaded to the Chrome Developer Dashboard](#for-an-extension-uploaded-to-the-chrome-developer-dashboard).
 
 > [!NOTE]
```

Comment:
At least one of these was swapped to a warning instead of a note. We should be consistent on which we use.

Reply:
Should we also add the note about using Plasmo here?

Reply:
```suggestion
> [!WARNING]
```